### DISCUSSION TITLE
章立てについて

### DISCUSSION THREAD

**Discussion by hideojoho** [ir100/ir100#8] [General]
## ゴール
- コンテンツの大まかな構造について議論し、暫定的に決定する

## 動機
- 構造を提示することで、共通意識を醸成したい
- 構造を提示することで、貢献先をイメージしやすくしたい

## たたき台
- https://github.com/ir100/ir100/wiki/%E7%AB%A0%E7%AB%8B%E3%81%A6%E6%A1%88

**Comment by tmanabe** [comment]
いま、なんとなく処理順に並んでいる気がしますが、
索引付け、順位付け、オフライン評価、あたりが情報検索のコアな気がするので、このへんを冒頭に持ってきて、
日本語前処理、クローリング、あたりは発展編でも良さそうな気がします。

あと、発展編に正解のアノテーションの話もあったら良いと思います（オフライン評価に含むかもしれないですが）

**Reply by hideojoho** [reply]
ありがとうございます！修正してみました。
https://github.com/ir100/ir100/wiki/%E7%AB%A0%E7%AB%8B%E3%81%A6%E6%A1%88/_compare/ea7e86e37798f23e3e43a831da5f97ecee3a5ab8

**Comment by hideojoho** [comment]
思いついた改善点
- [ ] Efficiencyに関するお題を入れる章がない・不明確
- [ ] QACの他にクエリ提案・クエリ拡張もあるので、どこかにまとめたい

**Reply by tmanabe** [reply]
Efficiency に関しては、索引付けのところで postings list を辿るときに、
skip list の話とか WAND（順位付けと密結合ですが……）の話とかはあっても良いのかなと思いますね。

**Comment by hideojoho** [comment]
## Miroボードの設置

- 参加の敷居を下げるために[Miroボード](https://miro.com/app/board/uXjVOYlOT5M=/?invite_link_id=458656977449)を設置してみたのですが、いかがでしょうか。
- ~~パスワードは「ir100000」です。~~ →取り払いました。
